src/components/ControlPanel.jsx

```diff
@@ -49,12 +51,19 @@ class ControlPanel extends React.Component {
   }
 
   toggleFieldGuide() {
-    if (this.props.dialog) {
+    if (this.props.dialogComponent === 'FieldGuide') {
```

Comment:
I have to doublecheck the code to see what `dialogComponent` is for, but it might be worth turning these string into `const TYPE_FIELDGUIDE = 'FieldGuide';`, simply to avoid any potential muckups like `dialogComponent === 'fieldguide'` causing the code not to realise what you're talking about.